doc/news/changes/minor/20200127PeterMunch-1

```diff
@@ -0,0 +1,4 @@
+New: ParameterHandler::parse_input_from_json and  ParameterHandler::parse_input_from_xml can skip
```

Comment:
```suggestion
New: ParameterHandler::parse_input_from_json() and  ParameterHandler::parse_input_from_xml() can skip
```
We try to consistently use the braces in function names (even though they are not necessary for doxygen to resolve them).